src/components/specific/models/models-manager/models-table/model-actions-cell/ModelActionsCell.vue

```diff
@@ -97,6 +97,16 @@
             {{ $t("ModelActionsCell.archiveButtonText") }}
           </template>
         </BIMDataButton>
+        <template v-if="model.type === MODEL_TYPE.META_BUILDING">
+          <BIMDataButton
+            class="model-actions-cell__menu__btn"
+            ghost
+            squared
+            @click="onClick('edit')"
```

Comment:
As this is a Meta Building specific action I think we could rename the event to something like `"edit-metaBuilding"`.
This would clarify event purpose.